history/history-network.md

```diff
@@ -200,6 +207,40 @@ content          = SSZ.serialize(block_header_with_proof)
 content_key      = selector + SSZ.serialize(block_number_key)
 ```
 
+##### Ephemeral Block Headers
+
+This content type represents block headers *near* the HEAD of the chain.  They are provable by tracing through the chain of `header.parent_hash` values.  All nodes in the network are assumed to store some amount of this content.  The `Ping.custom_data` and `Pong.custom_data` fields can be used to learn the number of recent headers that a client makes available.
+
+> Note: The history network does not provide a mechanism for knowing the HEAD of the chain. Clients to this network **MUST** have an external oracle for this information.  The Portal Beacon Network is able to provide this information.
+
+> Note: The content-id for this data type is not meaningful.
+
+> Note: This message is not valid for Gossip.  Clients **SHOULD** not send or accept gossip messages for this content type.
+
+> Note: Clients **SHOULD** implement a mechanism to purge headers older than 8192 blocks from their content databases.
+
+```python
+# Content and content key
+
+ephemeral_headers_key = Container(block_hash: Bytes32, ancestor_count: uint8)
```

Comment:
nit: Should we explicitly declare that `ancestor_count` can't be bigger than `MAX_EPHEMERAL_HEADER_PAYLOAD`? Or is it enough that it's implied by the content type?

Or is it ok to request more than 256, but answer shouldn't have more than that?

Reply:
I'll make these things explicitly stated.  They are enforced by the encoding types but worth making them stated limits too.